Arteon.Core/Entities/Room.cs

```diff
@@ -1,24 +1,29 @@
 ﻿using Arteon.Core.Common.Enums;
+using System.ComponentModel.DataAnnotations;
 
 namespace Arteon.Core.Entities
 {
     public class Room : BaseEntity
     {
-        public Room(RoomType type, int countOfPerson, DateTime? lockStart, DateTime? lockEnd, Guid? id = null)
+        public Room(RoomType typeId, int occupacity, int roomNumber,  Guid? id = null, double pricePerDay = 0)
```

Comment:
Why typeId?
RoomType is Enum
Rename to Type

Reply:
Done
TypeId was renamed to Type.